backend/mora/service/facet.py

```diff
@@ -73,7 +73,8 @@ def list_facets(orgid):
 
     :>jsonarr string name: The facet name.
     :>jsonarr string path: The location on the web server.
-    :>jsonarr string desc: Human readable description.
+    :>jsonarr string user_key: Human readable description.
```

Comment:
“Short, unique key identifying the facet.” — we use this internally for looking up the facet.